pkgs/top-level/all-packages.nix

```diff
@@ -25048,4 +25048,5 @@ in
 
   keycard-cli = callPackage ../tools/security/keycard-cli {};
 
+  streamlit = callPackage ../applications/science/machine-learning/streamlit { };
```

Comment:
can you make an attempt to move this to a somewhat alphabetized place :)

Reply:
I can try. That file itself is not exactly alphabetized, so I am not sure what would the best place. Suggestions?

Reply:
yep, it's not, but some where that roughly makes sense.... I think most people would like to avoid just a completely random listing.

there's a steamlink package, probably after that